src/components/new-request/search.vue

```diff
@@ -291,7 +291,7 @@
             :href="colinLink"
             target="_blank"
           >
-            Go to Corporate Online to {{ isConversion ? 'Alter' : 'Register' }}
+            Go to Corporate Online to {{ isConversion ? 'Alter' : isRestoration ? 'Restore' : 'Register'}}
```

Comment:
Please change this to a computed (get) and also include the "Change Name" option.

Reply:
Fixed, thanks.